backend/codesirius/problems/serializers/reference_solution.py

```
@@ -0,0 +1,62 @@
+        error_messages={
+            "required": "The language field is required.",
+            "null": "The language field must not be null.",
+        },
+    )
+    verdict = serializers.ChoiceField(
+        read_only=True,
+        choices=ReferenceSolution.Verdict.choices,
+        default=ReferenceSolution.Verdict.PENDING,
+    )
+
+    def create(self, validated_data: Dict) -> ReferenceSolution:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)